quantum/mousekey.c

```diff
@@ -196,7 +196,7 @@ static uint8_t move_unit(void) {
         speed = mousekey_accel & (1 << 2) ? mk_accelerated_speed : mk_decelerated_speed;
     } else if (mousekey_repeat && mouse_timer) {
         const uint16_t time_elapsed = timer_elapsed(mouse_timer) / 50;
-        speed                       = mk_initial_speed + MOUSEKEY_MOVE_DELTA * time_elapsed + MOUSEKEY_MOVE_DELTA * 0.5 * time_elapsed * time_elapsed;
+        speed                       = mk_initial_speed + MOUSEKEY_MOVE_DELTA * time_elapsed + (MOUSEKEY_MOVE_DELTA / 2) * time_elapsed * time_elapsed;
```

Comment:
provided my mathing is right, I'd opt for this instead, since it's less likely to hit fractions, outright. 
```suggestion
        speed                       = mk_initial_speed + MOUSEKEY_MOVE_DELTA * time_elapsed + (MOUSEKEY_MOVE_DELTA * time_elapsed * time_elapsed) / 2;
```

Reply:
This proposed change will result with a slightly larger output firmware (with `LTO`).